### DISCUSSION TITLE
Resume review x ATS score

### DISCUSSION THREAD

**Discussion by Dhairy2003** [recodehive/resume-pitch#90] [Q&A]
![dhairya .pdf](https://github.com/user-attachments/files/17005754/dhairya.pdf)

LinkedIn: https://www.linkedin.com/in/dhairy?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app

**Comment by sanjay-kv** [comment]
[dh.pdf](https://github.com/user-attachments/files/17089267/dh.pdf)

[ATS Version of CV (22).docx](https://github.com/user-attachments/files/17089268/ATS.Version.of.CV.22.docx)

@Dhairy2003
